rmgpy/reaction.py

```diff
@@ -1101,18 +1104,33 @@ def is_balanced(self):
         for reactant in self.reactants:
             if isinstance(reactant, Species):
                 molecule = reactant.molecule[0]
+                for atom in molecule.atoms:
+                    if not isinstance(atom, CuttingLabel):
+                        reactant_elements[atom.element] += 1
             elif isinstance(reactant, Molecule):
                 molecule = reactant
-            for atom in molecule.atoms:
-                reactant_elements[atom.element] += 1
-
+                for atom in molecule.atoms:
+                    if not isinstance(atom, CuttingLabel):
+                        reactant_elements[atom.element] += 1
```

Comment:
Is it possible for a molecule object to have vertices that are cutting labels? We could get rid of this check here if not

Reply:
no, molecule object can not have cutting labels. Since cutting label does not have element attribute, this check is required to avoid error.